app/main.py

```diff
@@ -104,44 +103,15 @@ def start_speech_recognition():
         pass
 
 
-p = pyaudio.PyAudio()
-
-stream = None
-
-
-def open_stream():
-    global stream
-    stream = p.open(format=pyaudio.paInt16, channels=1, rate=16000, input=True, frames_per_buffer=20480)
-    stream.start_stream()
-    print("--------------- RECOGNITION STARTED -----------------")
-
-
-def close_stream():
-    global stream
-    stream.stop_stream()
-    stream.close()
-
-
 websocketThread.start()
 
-open_stream()
+liveSpeech = LiveSpeech(lm=False, keyphrase='susi', kws_threshold=1e-15)
 
-# TODO: Decide threshold by a training based system.
-# adjust threshold manually for now.
-sphinxRecognizer = SphinxRecognizer(threshold=1e-30)
+for phrase in liveSpeech:
+    print(phrase)
```

Comment:
This does not affect much. This serves as additional logging output for the user.